docs/howto/manage-applications.md

```diff
@@ -267,7 +240,7 @@ See more: {ref}`scale-an-application-horizontally`
 ```
 
 
-````{dropdown} Expand to view an example featuring the machine charm for Wordpress
+````{dropdown} Example scenario: scaling Wordpress
```

Comment:
for later: replace`wordpress` as an example charm here. It cannot be deployed with juju 3.x as it does not support ubuntu@20.04 or newer.

Reply:
Any suggestion for an alternative?

Reply:
The data platform charms tend to be high quality. Maybe pick postgresql or mysql?